cmdget.go

```diff
@@ -99,10 +99,14 @@ func runGet1(args []string) error {
 			}
 			repl = repl1
 		}
-		// Automatically generate a go.mod file if one doesn't
-		// already exist, because otherwise the directory cannot be
-		// used as a module.
-		if err := ensureGoModFile(repl.modulePath, repl.dir); err != nil {
+		// Automatically generate a go.mod file if one doesn't already exist,
+		// because otherwise the directory cannot be used as a module. Remebering
```

Comment:
s/Remebering/Remembering/

Reply:
Thanks. I can't be trusted with spelling, let alone a spell-checker! 🦆